webapp/routes/route_team.jsx

```diff
@@ -147,6 +152,63 @@ function onPermalinkEnter(nextState, replace, callback) {
     );
 }
 
+function onChannelByIdentifierEnter(state, replace, callback) {
```

Comment:
Can you add a comment here to explain that the identifier may either be:
- A username that starts with an @ sign
- An email containing an @ sign

The logic here didn't really make sense until I read through the discussion here. I'll update the ticket to reflect the discussion here since the ticket has it without the at signs